packages/api/src/external/commonwell/cq-bridge/coverage-enhancer-factory.ts

```diff
@@ -0,0 +1,36 @@
+import { CookieManagerOnSecrets } from "@metriport/core/domain/auth/cookie-management/cookie-manager-on-secrets";
+import { CoverageEnhancer } from "@metriport/core/external/commonwell/cq-bridge/coverage-enhancer";
+// import { CoverageEnhancerCloud } from "@metriport/core/external/commonwell/cq-bridge/coverage-enhancer-cloud";
+import { CommonWellManagementAPI } from "@metriport/core/external/commonwell/management/api";
+import { Config } from "../../../shared/config";
+import { PatientUpdaterCommonWell } from "../patient-updater-commonwell";
+import { CoverageEnhancerApiLocal } from "./coverage-enhancer-api-local";
+
+export function makeCoverageEnhancer(): CoverageEnhancer | undefined {
+  // if (Config.isCloudEnv()) {
+  //   const cwPatientLinkQueueUrl = Config.getCWPatientLinkQueueUrl();
+  //   if (!cwPatientLinkQueueUrl) {
+  //     console.log(`Could not return a CoverageEnhancer, mising cwPatientLinkQueueUrl`);
+  //     return undefined;
+  //   }
+  //   return new CoverageEnhancerCloud(Config.getAWSRegion(), cwPatientLinkQueueUrl);
+  // }
```

Comment:
is there a reason this is commented out? assuming b/c the EC on lambda is still a TODO?

Reply:
yeah, lambda related... there's a bunch of commented out code that I'll have to clean up once this is done